### DISCUSSION TITLE
Ender 3 v2 BL Touch - Does M420 S1 use the mesh saved to EEPROM?

### DISCUSSION THREAD

**Discussion by synchrotheus** [Jyers/Marlin#1399] [Q&A] [answered]
My gcode is as follows:

G92 E0 ; reset extrusion distance
G28 ; home all axes
M420 S1 ; use saved level
G1 X5 Y10 Z0.2 F3000 ; get ready to prime
G1 X30 Y15 F1500 ; avoid binder clips
G1 X160 E15 F600 ; prime nozzle
G1 X180 F5000 ; quick wipe

I'm using the 3x3 (not high-speed) Jyers firmware. Does the M420 command use the created mesh saved on EEPROM? I've done a few bed level prints and most of them level properly but the bottom left of the print bed always seems to be too close to the nozzle. It feels as if the printer isn't using the BL Touch or the created mesh when printing objects.

**Comment by Jyers** [comment, marked as answer]
Yes, M420 S1 will tell the printer to use the mesh. If it's still not working you might have another issue. I'd recommend looking through other discussions to see if you can identify something that might help.
